src/admin-data-import/components/AdminDataImportPage/AdminDataImportPage.jsx

```diff
@@ -102,6 +114,14 @@ const AdminDataImportPage = () => {
 
     return (
         <>
+            {
+                isModalOpen &&
+                    <ImportSummaryModal
+                        isOpen={true}
+                        results={importModalContent}
+                    />
+            }
+
```

Comment:
Have you considered using `isOpen={isModalOpen}` instead? Is conditional rendering necessary in this case?